frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -117,7 +117,7 @@ export default function SideNavigation({
               selectedPage={page}
               primaryText='Marketplace'
               drawerOpen={drawerOpen}
-              menuPage='marketplace'
+              menuPage='model'
```

Comment:
This should match when we're at `<baseUrl>/`, not when we're at `<baseUrl>/model`. Might be as simple as making this an empty string.